stock_move_backdating/__openerp__.py

```diff
@@ -1,6 +1,6 @@
 # -*- coding: utf-8 -*-
-# ©2015 Agile Business Group (<http://www.agilebg.com>)
-# ©2015 BREMSKERL-REIBBELAGWERKE EMMERLING GmbH & Co. KG
+# © 2016 Agile Business Group (<http://www.agilebg.com>)
```

Comment:
I think it is better to keep it 2015 or 2015-2016 (in case of copyright claims you want to state that you were already copyright holder in 2015 right?)
Remove the (c) signs (nice to have)